src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -7411,6 +7462,28 @@ Object {
   "functions": Array [],
   "name": "Form",
   "properties": Array [
+    Object {
+      "description": "",
+      "inlineType": Object {
+        "name": "AnalyticsMetadata",
```

Comment:
In the other components, the type was prefixed with the component name, but here for Form it is jus `AnalyticsMetadata` (same in line 16662). Is this intended? Could it cause problems? We could probably prevent it by moving these into a `Form` namespace

Reply:
Moved